caffe2/serialize/inline_container.h

```diff
@@ -59,57 +63,68 @@ namespace torch { namespace jit {
 //         been written. We place the variable-length index at the end and do
 //         not put any indicies into the header to fulfill this constraint.
 
+// The serialized model, which contains all the metadata information,
+// should be stored as the last record. Because the size of tensor data is
+// usually stable. As long as the shape and type of the tensor do not change,
+// the size of the data won't change. On the other sied, the size of the
+// serialized model is likely to change, so we store it as the last record, and
+// we don't need to move previous records when updating the model data.
+
 namespace {
-  struct RecordTags {
-    enum {
-      STORAGE = 1,
-      FOOTER = 2,
-    };
+
+struct RecordTags {
+  enum {
+    STORAGE = 1,
```

Comment:
I'd suggest to use kStorage and kFooter as enum names.
https://google.github.io/styleguide/cppguide.html#Enumerator_Names